Various manual pages changes:
* removes LIBEXECDIR from slapd.8
* removes references to non-existing manpages (bz 624616)

--- openldap-2.4.24.orig/doc/man/man1/ldapmodify.1
+++ openldap-2.4.24/doc/man/man1/ldapmodify.1
@@ -364,9 +364,7 @@ exit status and a diagnostic message bei
 .BR ldap_add_ext (3),
 .BR ldap_delete_ext (3),
 .BR ldap_modify_ext (3),
-.BR ldap_modrdn_ext (3),
-.BR ldif (5),
-.BR slapd.replog (5)
+.BR ldif (5)
 .SH AUTHOR
 The OpenLDAP Project <http://www.openldap.org/>
 .SH ACKNOWLEDGEMENTS
--- openldap-2.4.24.orig/doc/man/man5/ldap.conf.5
+++ openldap-2.4.24/doc/man/man5/ldap.conf.5
@@ -317,6 +317,7 @@ certificates in separate individual file
 .B TLS_CACERT
 is always used before
 .B TLS_CACERTDIR.
+The specified directory must be managed with the OpenSSL c_rehash utility.
 This parameter is ignored with GnuTLS.
 
 When using Mozilla NSS, <path> may contain a Mozilla NSS cert/key
--- openldap-2.4.24.orig/doc/man/man5/ldif.5
+++ openldap-2.4.24/doc/man/man5/ldif.5
@@ -270,8 +270,7 @@ commands.
 .BR ldapmodify (1),
 .BR slapadd (8),
 .BR slapcat (8),
-.BR slapd\-ldif (5),
-.BR slapd.replog (5).
+.BR slapd\-ldif (5).
 .LP
 "LDAP Data Interchange Format," Good, G., RFC 2849.
 .SH ACKNOWLEDGEMENTS
--- openldap-2.4.24.orig/doc/man/man5/slapd-config.5
+++ openldap-2.4.24/doc/man/man5/slapd-config.5
@@ -2068,7 +2068,6 @@ default slapd configuration directory
 .BR slapd.conf (5),
 .BR slapd.overlays (5),
 .BR slapd.plugin (5),
-.BR slapd.replog (5),
 .BR slapd (8),
 .BR slapacl (8),
 .BR slapadd (8),
--- openldap-2.4.24.orig/doc/man/man5/slapd.conf.5
+++ openldap-2.4.24/doc/man/man5/slapd.conf.5
@@ -1998,7 +1998,6 @@ default slapd configuration file
 .BR slapd.backends (5),
 .BR slapd.overlays (5),
 .BR slapd.plugin (5),
-.BR slapd.replog (5),
 .BR slapd (8),
 .BR slapacl (8),
 .BR slapadd (8),
--- openldap-2.4.24.orig/doc/man/man8/slapd.8
+++ openldap-2.4.24/doc/man/man8/slapd.8
@@ -5,7 +5,7 @@
 .SH NAME
 slapd \- Stand-alone LDAP Daemon
 .SH SYNOPSIS
-.B LIBEXECDIR/slapd 
+.B slapd 
 [\c
 .BR \-4 | \-6 ]
 [\c
@@ -301,7 +301,7 @@ the LDAP databases defined in the defaul
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd
+	slapd
 .ft
 .fi
 .LP
@@ -312,7 +312,7 @@ on voluminous debugging which will be pr
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-f /var/tmp/slapd.conf \-d 255
+	slapd -f /var/tmp/slapd.conf -d 255
 .ft
 .fi
 .LP
@@ -320,7 +320,7 @@ To test whether the configuration file i
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-Tt
+	slapd -Tt
 .ft
 .fi
 .LP
